apps/chat/src/types/common.ts

```diff
@@ -26,7 +26,8 @@ export enum BackendResourceType {
 }
 
 export interface EntityPublicationInfo {
-  action: PublishActions;
+  action?: PublishActions;
+  isDeleted?: boolean;
```

Comment:
we already have `PublishActions.DELETE`.
Why do we need `isDeleted` here?

Reply:
it how I marked entities which have been unpublished. Renamed to `inNotExist`